include/opencensus.hrl

```diff
@@ -24,8 +24,8 @@
          }).
 
 -record(span, {
-          start_time  :: opencensus:time_us(),
-          end_time  :: opencensus:time_us() | undefined,
+          start_time  :: wts:timestamp() | opencensus:time_us(),
+          end_time  :: wts:timestamp() | opencensus:time_us() | undefined,
```

Comment:
Why is opencensus:time_us still allowed here?

Reply:
Actually the incorrect part is the `ws:timestamp()` for `end_time` :). When a span is finished the start time and end time are converted to `time_us()`.

Reply:
hmm using the same field for 2 different types feels a bit dirty, it means dialyzer will not catch errors when something gets falsely set to a _us ahead of time. How do you feel about making those separated fields?

Reply:
Hm, just realized as I am adding support for reporting to Stackdriver that collectors take time in RFC3339 format, not simply an integer. So I guess converting here from native isn't worth it and can just do it at time of reporting since it may have to convert to something else anyway.